### ISSUE TITLE
onSave not triggered if value evaluate to false

### ISSUE THREAD

**Issue by ockam** [jollyjerr/react-autosave#12]
In the `useAutosave.tsx`, when evaluating if onSave will be called, if the value is falsy (empty string, 0), `onSave` is not called.
```
    if (initialRender.current) {
      initialRender.current = false;
    } else if (debouncedValueToSave) {
      onSave(debouncedValueToSave);
    }
```
You should remove this check.

P.S. thanks for this great hook!

**Comment by jollyjerr** [comment]
This is great, thank you! I'll let you know when it's patched 🙇🏼

**Comment by jollyjerr** [comment]
closed by #14

**Comment by jollyjerr** [comment]
Patched in [0.4.0](https://www.npmjs.com/package/react-autosave/v/0.4.0) 🎉
